packages/create-react-native-library/templates/native-view-legacy/src/{%- project.name %}ViewNativeComponent.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  requireNativeComponent,
+  UIManager,
+  Platform,
+  ViewStyle,
+} from 'react-native';
+
+const LINKING_ERROR =
+  `The package '<%- project.slug -%>' doesn't seem to be linked. Make sure: \n\n` +
+  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
+  '- You rebuilt the app after installing the package\n' +
+  '- You are not using Expo Go\n';
+
+type <%- project.name -%>Props = {
+  color: string;
+  style: ViewStyle;
+};
+
+const ComponentName = '<%- project.name -%>View';
+
+export default UIManager.getViewManagerConfig(ComponentName) != null
```

Comment:
Generally good to avoid default exports

```suggestion
export const <%- project.name -%>View = UIManager.getViewManagerConfig(ComponentName) != null
```